cfme/tests/control/test_basic.py

```diff
@@ -463,13 +467,35 @@ def test_control_assign_actions_to_event(request, policy, action):
 
 
 @pytest.mark.tier(3)
-def test_assign_condition_to_control_policy(request, policy_and_condition):
+@pytest.mark.uncollectif(lambda appliance, policy_and_condition: (
+    policy_and_condition.policy is policies.PhysicalInfrastructureControlPolicy and
+    appliance.version < "5.9"))
+@pytest.mark.parametrize("policy_and_condition", POLICIES_AND_CONDITIONS,
+                         ids=lambda item: item.name)
+def test_assign_condition_to_control_policy(request, policy_and_condition, condition_collection,
+                                            policy_collection):
     """This test checks if a condition is assigned to a control policy.
     Steps:
         * Create a control policy.
         * Assign a condition to the created policy.
     """
-    policy, condition = policy_and_condition
+    condition_class = policy_and_condition.condition
```

Comment:
**REQUIRED:** The policy and condition here shouldn't be created inside the test. These are fixtures I believe and should be created as such outside. I believe we already have tests that test creating conditions and policies.

Reply:
Agree, but there is a problem with uncollecting. Here is the problematic place https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/control/test_basic.py#L368. I can't create a Physical Infrastructure policy in CFME 5.8.

Reply:
I think I would prefer to do it in this way, put those into a fixture and skip if we get those values, now, I KNOW, that we shouldn't do that, in that a skip is for a transient failure due to some other factor, but I think having the test body represent what it _should_ actually test, is more important right now.